src/buckets/components/createBucketForm/BucketOverlayForm.tsx

```diff
@@ -61,30 +65,39 @@ interface Props {
   measurementSchemaList?: typeof MeasurementSchemaList
   showSchemaValidation?: boolean
 }
-
+// todo: determine exact real type of updates?  or don't bother
+// bc the specified return type is not exactly what we get back
+// (also get back bucketID and orgID, both of which I am using)
 interface State {
   showAdvanced: boolean
   schemaType: 'implicit' | 'explicit'
   newMeasurementSchemas: typeof MeasurementSchemaCreateRequest[]
+  measurementSchemaUpdates: any[]
```

Comment:
Why not make a new type that includes the bucketID and orgID?

Reply:
because there is already a type with that, it was fixed by stuart.  this todo is done/addressed already. (see the most recent code)

Reply:
Ok, then can we remove the comment since it has been dealt with?